### ISSUE TITLE
Add support for VSCodium

### ISSUE THREAD

**Issue by raguay** [foambubble/foam#26]
It will not run on vscodium, the freeware version without Microsoft trackers.

**Comment by jevakallio** [comment]
@raguay Thanks for the issue report. What error are you seeing?

**Comment by madeindjs** [comment]
You may want to add this extension to [Open VSX](https://github.com/eclipse/openvsx) to make it available for [VSCodium](https://vscodium.com/) (wich is a VSCode version without Microsoft licence and trackers).

> Open VSX is a [vendor-neutral](https://projects.eclipse.org/projects/ecd.openvsx) open-source alternative to the [Visual Studio Marketplace](https://marketplace.visualstudio.com/vscode). It provides a server application that manages [VS Code extensions](https://code.visualstudio.com/api) in a database, a web application similar to the VS Code Marketplace, and a command-line tool for publishing extensions similar to [vsce](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#vsce). - [eclipse/openvsx: An open-source registry for VS Code extensions](https://github.com/eclipse/openvsx)

To publish it, you simply have to [follow this guide](https://github.com/eclipse/openvsx/wiki/Publishing-Extensions).

**Comment by raguay** [comment]
The errors are that it can't download from Marketplace. I'll try the fix
that Alexandre proposes to see if it helps.

**Comment by jevakallio** [comment]
Thanks! I'll look into uploading foam-vscode to openvsx, but at the moment we depend on 5 other [recommended extensions](https://foambubble.github.io/foam/recommended-extensions) which need to first be uploaded there too, so this might be a bit difficult to orchestrate.

**Comment by GiantZOC** [comment]
I am a vscodium user and am trying out Foam today.  I did some research to see which projects were on openvsx and which ones are not.  Here is what I found about the recommended extensions for the template I installed.

Available on openvsx
- Markdown All in One: http://open-vsx.org/extension/yzhang/markdown-all-in-one
- Git Lens: http://open-vsx.org/extension/eamodio/gitlens
- Prettier: http://open-vsx.org/extension/esbenp/prettier-vscode

Not Available
- Markdown Links Issue: Issue Created Already https://github.com/tchayen/markdown-links/issues/11
- VS Code Markdown Notes: Issue Created Already https://github.com/kortina/vscode-markdown-notes/issues/41
- Grey Matter: Not available but not going to request it since it is just a theme.

I was able to get my environment working by downloading the extensions from the VSCode Marketplace and then installing them manually.

**Comment by notpushkin** [comment]
It is my understanding that if some extension isn't available on Open VSX yet, _anybody_ can upload it (and the author of the extension can claim ownership over a namespace / publisher afterwards). I'm not sure if it is ethical to do so, but it could be a valid workaround.

**Comment by SamGuay** [comment]
Although not all [recommend extensions](https://foambubble.github.io/foam/recommended-extensions) are currently available on Open VSX, it is possible to run Foam with VSCodium @raguay. I've freshly installed Ubuntu 20.04 and after fixing the [Marketplace issue by following this temporary workaround](https://github.com/VSCodium/vscodium/issues/418#issuecomment-643664182) on VSCodium, I was able to install all of them from the Marketplace.

Thanks for Foam! I tested Roam and was happily surprised! The open-source flavour is what I was looking for, especially after playing with [Joplin](https://joplinapp.org/) for while. I'll investigate for a way to easily sync & encrypt my notes.
